bip-0159.mediawiki

```diff
@@ -13,15 +13,16 @@
 
 == Abstract ==
 
-Define a service bit that allow pruned peers to signal their limited services
+Define a service bit that allows pruned peers to signal their limited services.
 
 ==Motivation==
 
-Pruned peers can offer the same services as traditional peer except of serving all historical blocks.
-Bitcoin right now only offers the NODE_NETWORK service bit which indicates that a peer can serve
+Pruned peers can offer the same services as traditional peers, except that of serving all historical blocks.
+Bitcoin right now only offers the <code>NODE_NETWORK</code> service bit to indicate that a peer can serve
 all historical blocks.
-# Pruned peers can relay blocks, headers, transactions, addresses and can serve a limited number of historical blocks, thus they should have a way how to announce their service(s)
-# Peers no longer in initial block download should consider connecting some of its outbound connections to pruned peers to allow other peers to bootstrap from non-pruned peers
+# Pruned peers can relay blocks, headers, transactions, and addresses, but they guarantee serving only a minimum limited number of historical blocks; thus, they should have a way to announce their service(s)
+# Pruned peers are defined by this BIP in service bit terms as those having <code>NODE_NETWORK_LIMITED</code> set and any service bit that signals serving the complete block chain, like <code>NODE_NETWORK</code>, not set
```

Comment:
I think this is better but still hard to read.
Maybe we should describe the expected behavior when both service bits (`NODE_NETWORK` and `NODE_NETWORK_LIMITED`) are set?

Reply:
Good point. Rewrote it, and moved it out of the motivation section into the specification instead.